django/forms/widgets.py

```diff
@@ -48,6 +48,7 @@
     "SplitDateTimeWidget",
     "SplitHiddenDateTimeWidget",
     "SelectDateWidget",
+    "ChoiceWidget",
```

Comment:
`ChoiceWidget` is not a documented widget. I'd not add it to `__all__`.

Reply:
Sure, I only added this because, I was getting `'ChoiceWidget' is not declared in __all__`. I wasn't fully aware of purpose of `__all__`.